resources/languages/ja_JP.yaml

```diff
@@ -11,15 +11,15 @@ player:
 command:
   not-a-player: このコマンドを実行するにはプレイヤーでなければなりません。
   notifications:
-    usage: '/{command}'
+    usage: '/<command>'
     output:
       no-notifications: 未読通知はありません。
       click-to-open: クリックするとウェブブラウザで開きます
       fail: 通知一覧を取得しようとした際にエラーが発生しました。この問題については、サーバー管理者に連絡してください。
       notification: <hover:show_text:"クリックしてウェブブラウザで開く"><click:open_url:"<url>"><message>
     description: ウェブサイトの通知一覧を表示します。
   register:
-    usage: '/{command} <メールアドレス>'
+    usage: '/<command> \<username> \<email>'
```

Comment:
@ViaSnake Thanks again! Minor note, when escaped with a backslash, these placeholders should be translated like they were before.

Reply:
> @ViaSnake Thanks again! Minor note, when escaped with a backslash, these "placeholders" are not actually placeholders and should be translated like they were before. Only `<command>` is a placeholder here.

Thanks for letting me know! I have corrected translations.